src/commands/ext-install.ts

```diff
@@ -54,6 +54,146 @@ interface InstallExtensionOptions {
   force?: boolean;
 }
 
+/**
```

Comment:
Consider moving this above InstallExtensionOptions too, so that InstallExtensionOptions is next to installExtension (or just move InstallExtensionOptions to be an inline args type)

Reply:
Good call, done